cs/cli/vw_clr.vcxproj

```diff
@@ -48,7 +48,7 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <Import Project="$(ProjectDir)\..\..\vowpalwabbit\Build.props" />
   <PropertyGroup Label="Globals">
-    <WindowsTargetPlatformVersion>10.0.10240.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
```

Comment:
Please make sure to set these before importing Microsoft.Cpp[.Default].props (also on output line 33), otherwise it will (for a few operations) default to 8.1 and can fail in the case where 8.1 is not installed on the system.